src/main/groovy/io/openliberty/tools/gradle/utils/ArtifactDownloadUtil.groovy

```diff
@@ -40,13 +39,14 @@ public class ArtifactDownloadUtil {
     }
 	
 	public static File downloadSignature(Project project, String groupId, String artifactId, String type, String version, File esa) throws PluginExecutionException {
-		String coordinates = groupId + ":" + artifactId + ":" + version + "@" + type
+        String coordinates = groupId + ":" + artifactId + ":" + version + "@" + type
 		def dep = project.dependencies.create(coordinates)
 		def config = project.configurations.detachedConfiguration(dep)
 		def sig = downloadFile(project, config, coordinates);
 		//if signature and esa file are not in same directory, copy signature file to esa parent directory.
 		if (!sig.getParent().equals(esa.getParent())) {
-			GFileUtils.copyFile(sig, new File(esa.getAbsolutePath() + ".asc"))
+            project.getLogger().debug("Copying " + sig + " to esa.getAbsolutePath()" + ".asc")
```

Comment:
I think you meant to have `esa.getAbsolutePath()` actually resolve here. You have it inside quotation marks.